Sources/Jaeger/Jaeger/Agent/CoreData/CoreDataSpan+CoreDataClass.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CoreDataSpan+CoreDataClass.swift
+//  Jaeger
+//
+//  Created by Simon-Pierre Roy on 11/6/18.
+//
+//
+
+import Foundation
+import CoreData
+
+/// Auto-generated class to represent a `CoreDataSpan`.
+@objc(CoreDataSpan)
+class CoreDataSpan: NSManagedObject {
+    
+    /**
+     Creates and adds a new `CoreDataSpan` in a given `NSManagedObjectContext`.
+     
+     - Parameter context: The context to which the span will be added.
+     - Parameter startTime: The start time of the span.
+     - Parameter data: The binary representation of the span.
+     - Returns: A new `CoreDataSpan` inserted in the given `NSManagedObjectContext`.
+     */
+    @discardableResult
+    static func insertNewSpan(in context: NSManagedObjectContext, startTime: Date, data: Data) -> CoreDataSpan {
```

Comment:
Since the class name is called `CoreDataSpan` and this static function is creating a new instance of the class, does it make sense to rename `static func insertNewSpan(in context: NSManagedObjectContext, startTime: Date, data: Data) -> CoreDataSpan` to `static func create(in context: NSManagedObjectContext, startTime: Date, data: Data) -> CoreDataSpan`

Reply:
Yes it does! I will change that!